Modules/mathmodule.c

```diff
@@ -2016,6 +2016,78 @@ math_factorial(PyObject *module, PyObject *arg)
 }
 
 
+/*[clinic input]
+math.lcm
+
+	x as a: object
+	y as b: object
+    /
+least common multiple of x and y
+[clinic start generated code]*/
+
+static PyObject*
+math_lcm(PyObject*module,PyObject*a,PyObject*b)
+{
+    PyObject * g,*m,*f,*ab;
```

Comment:
What's happening?

Reply:
This is my first contribution,can you tell me what's the problem here?

Reply:
Please follow PEP7 and code formatting